Lib/test/test_gc.py

```diff
@@ -822,6 +822,68 @@ def test_get_objects_arguments(self):
         self.assertRaises(TypeError, gc.get_objects, "1")
         self.assertRaises(TypeError, gc.get_objects, 1.234)
 
+    def test_38379(self):
```

Comment:
Maybe add a comment here explaining what the test is about.  "test_38379" will be a bit mysterious a few years from now.

Reply:
Absolutely - working with one finger goes slow 😉